### PR TITLE
Add switch languages shortcut

### LINKED ISSUE #169: [Feature request] Add hotkey to switch languages
Like Ctrl+Shift+S in Google Translate. I find this hotkey very useful

### PR DESCRIPTION
**Pull request by xfgusta**
Closes #169

### CODE REVIEW

File: dialect/main.py [hunk 1]
```diff
@@ -121,6 +121,11 @@ def setup_actions(self):
         about_action.connect('activate', self.on_about)
         self.add_action(about_action)
 
+        switch_action = Gio.SimpleAction.new('main', None)
```
Comment (rafaelmardojai) [inline dialect/main.py:124]: Name it `switch`.

Comment (rafaelmardojai) [inline dialect/main.py:124]: Name it switch.

**Review by rafaelmardojai — CHANGES_REQUESTED** [review]
Could you make `switch_btn` and `switch_btn2` in `window.ui` use the action?

With: `<property name="action-name">app.switch</property>`

https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/data/resources/window.ui#L189
https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/data/resources/window.ui#L721
 
And remove the buttons click callbacks on `window.py`:
https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/dialect/window.py#L305
https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/dialect/window.py#L319

And remove the `_button` attribute from `ui_switch`:
https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/dialect/window.py#L615

**Comment by xfgusta** [comment]
I'm so sorry, I forgot to add some files to commit :(

**Comment by rafaelmardojai** [comment]
> I'm so sorry, I forgot to add some files to commit :(

No problem, probably we're going to squash all on merge. :)

**Review by rafaelmardojai — APPROVED** [review]

**Comment by fushinari** [comment]
Just a minor suggestion, other than that, looks great!

File: dialect/main.py [hunk 2]
```diff
@@ -121,6 +121,11 @@ def setup_actions(self):
         about_action.connect('activate', self.on_about)
         self.add_action(about_action)
 
+        switch_action = Gio.SimpleAction.new('switch', None)
+        switch_action.connect('activate', self.on_switch)
```
[the remark this answers is not in the document]
Comment (xfgusta) [reply dialect/main.py:125]: When I try to pass `self.window.ui_switch` as an argument

```python
switch_action.connect('activate', self.window.ui_switch)
```

 I get this error:

```
AttributeError: 'NoneType' object has no attribute 'ui_switch'
```

I don't know how to solve this.

[the remark this answers is not in the document]
Comment (rafaelmardojai) [reply dialect/main.py:125]: We will need to move the call to `setup_actions` from `do_startup` to `do_activate` after the window creation:

From:
https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/dialect/main.py#L90
To:
https://github.com/dialect-app/dialect/blob/0b4895a09702478ae26b602b35ea535d40795364/dialect/main.py#L58

File: dialect/main.py [hunk 3]
```diff
@@ -56,6 +56,7 @@ def do_activate(self):
                 text=self.launch_text,
                 langs=self.launch_langs
             )
+        self.setup_actions()
```
Comment (rafaelmardojai) [inline dialect/main.py:59]:
```suggestion
            self.setup_actions()
```
Move it inside the window conditional.

**Comment by rafaelmardojai** [comment]
Thanks for working o this! Very appreciated :)

**Pull request merged**